apps/fulfillment-e2e/src/integration/fully-picking-a-picklist.cy.ts

```diff
@@ -0,0 +1,45 @@
+import { PickingFragment } from '../support/page_fragments/picking.fragment';
+
+describe('Fully pick a picking list', () => {
+  const pickingListId = '37cb241e-f18a-5768-985c-a2d7aff4875e';
+  const pickingFragment = new PickingFragment();
+
+  beforeEach(() => {
+    cy.visit(`/picking-list/picking/${pickingListId}`);
+  });
+
+  describe('Picking all product items', () => {
+    beforeEach(() => {
+      [pickingFragment.getProducts()].forEach((product) => {
+        const oryxCartQuantity = product.find('oryx-cart-quantity-input');
+        const increaseButton = oryxCartQuantity.find('[part="increase"]');
+
+        increaseButton.click({ multiple: true });
+      });
+
+      [pickingFragment.getProducts()].forEach((product) => {
+        const doneButton = product
+          .findByTestId('product-card-done')
+          .should('exist');
+
+        doneButton.click({ multiple: true });
+      });
+    });
+
+    it('must have all products on picked tab after done all of them', () => {
```

Comment:
I'm missing `Verify that the “Great job” illustration is shown and the “Finish picking” button appears` check here (we have it in ticket description)

Reply:
or is it `getAllItemsDonePlaceholder`?